### PR TITLE
Fix openQA config values and add info if it cannot trigger openqa

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: github-action.py [hunk 1]
```diff
@@ -920,17 +920,25 @@ def build(self):
...
                     openqa_client_path.parent.mkdir(parents=True, exist_ok=True)
                     with open(openqa_client_path, "w") as f:
-                        f.write(f"""[{server}]
+                        f.write(
+                            f"""[{server}]
 key = {key}
 secret = {secret}
-""")
+"""
+                        )
```
Comment (marmarek) [inline github-action.py]: Looking at it more, shouldn't qubes-infrastructure simply write this file instead?

**Review by marmarek — CHANGES_REQUESTED** [review]

File: github-action.py (hunk 1 above)
Comment (fepitre) [reply github-action.py]: Yes you are right, also, I would adapt the check with `openqa_client_path.exists()`.

File: github-action.py [hunk 2]
```diff
@@ -920,20 +920,11 @@ def build(self):
...
-                        f.write(f"""[{server}]
-key = {key}
-secret = {secret}
-""")
+                openqa_client_path = Path.home() / ".config/openqa/client.conf"
+                if all([openqa_client_path.exists(), OpenQA_Client, OpenQAClientError]):
                     try:
                         version = self.qubes_release.lstrip("r")
-                        client = OpenQA_Client(server=server)
+                        client = OpenQA_Client()
```
Comment (marmarek) [inline github-action.py:927]: Will it use correct server by default? Maybe just server name should stay?

Comment (fepitre) [reply github-action.py:927]: According to docstring https://github.com/os-autoinst/openQA-python-client/blob/main/src/openqa_client/client.py#L68, it should use the one we provide with `qubes-infrastructure`.

**Comment by marmarek** [comment]
PipelineRefresh

**Review by marmarek — APPROVED** [review]

**Pull request merged**